src/c++/vdo/base/io-submitter.c

```diff
@@ -386,8 +386,7 @@ void __submit_metadata_vio(struct vio *vio, physical_block_number_t physical,
 		return;
 	}
 
-	vdo_set_completion_callback(completion, process_vio_io,
-				    get_vio_bio_zone_thread_id(vio));
+	vdo_set_completion_callback(completion, vdo_submit_vio, get_vio_bio_zone_thread_id(vio));
 	vdo_launch_completion_with_priority(completion, get_metadata_priority(vio));
```

Comment:
You should not be changing the wrapping here.

Reply:
Modifying the wrapping to put it on one line was in Mike's change to "rename to submit_vio and submit_data_vio".